packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -75,15 +75,21 @@ export class TreeItem extends FASTElement {
     private itemsChanged(oldValue, newValue): void {
         if (this.$fastController.isConnected) {
             this.items.forEach((node: HTMLElement) => {
-                if (node instanceof TreeItem) {
+                if (isTreeItemElement(node)) {
                     // TODO: maybe not require it to be a TreeItem?
                     (node as TreeItem).nested = true;
                 }
             });
         }
     }
 
-    @observable
+    /**
+     * Tracks the nested state of the item
+     * @public
+     * @remarks
+     * HTML Attribute: nested
+     */
+    @attr({ mode: "boolean" })
     public nested: boolean;
```

Comment:
If this isn't intended to be set in HTML by a consumer, then probably better to make this into an `@observable`.

Reply:
I like these improvements - I’ll take a follow up here. Wanted to land a fix ASAP, sorry we missed you the first round!